app/Spago.hs

```diff
@@ -149,6 +149,7 @@ parser = do
 
     force   = CLI.switch "force" 'f' "Overwrite any project found in the current directory"
     verbose = CLI.switch "verbose" 'v' "Enable additional debug logging, e.g. printing `purs` commands"
+    noComments = CLI.switch "no-comments" 'C' "Strip comments in the generated package.dhall and spago.dhall files"
```

Comment:
It's kinda hard to come up with single letter switch name when most of the letters are taken :-) Do you have better suggestions than 'C'?

Reply:
I think `C` is great!